### PR TITLE
fix: Coordinator UI on front page

### LINKED ISSUE #324: Make co-ordinators name white and move to vue data
- [x] Black is not visible in teal background, make it white.
- [x] Also, since the names and profile pics of coordinators are dynamic, they shouldn't be hardcoded in HTML markup. Move to data section and loop over to render them
- [x] Use smaller standard text for coordinator names

### PR DESCRIPTION
**Pull request by Haider8**
fixes #324

### CODE REVIEW

File: pages/index.vue [hunk 1]
```diff
@@ -265,7 +231,21 @@ export default {
...
+          'https://lh6.googleusercontent.com/-Nabz6JJKsqc/AAAAAAAAAAI/AAAAAAAAAoI/biOxaDpjiec/photo.jpg'
+        ],
+        [
+          'Harshul Gupta',
+          'https://res.cloudinary.com/amuroboclub/image/upload/v1529768915/profile_img/IMG_20180328_004220805.jpg'
+        ],
+        [
+          'Kakul Shrivastava',
+          'https://res.cloudinary.com/amuroboclub/image/upload/v1543928884/profile_img/FB_IMG_1543928797162.jpg'
+        ]
```
Comment (iamareebjamal) [inline pages/index.vue]: Make it a list of object

```
{
  name: '',
  avatar: ''
}
```

What's the logic of making it a matrix?

Comment (Haider8) [reply pages/index.vue]: > What's the logic of making it a matrix?

Looks good

Should I make it a list of object?

Comment (iamareebjamal) [reply pages/index.vue]: Yes

Comment (iamareebjamal) [reply pages/index.vue]: Looks good, reads terrible

Comment (Haider8) [reply pages/index.vue]: Okay I will make it a list of object

**Pull request merged**